templates/data.html

```diff
@@ -30,7 +30,7 @@
 			<h4>Site Wide</h4>
 			<p>Datasets that power the entire pulse.cio.gov website.</p>
 				<ul><li>
-				Domain Inspect Results - <a href="https://s3.amazonaws.com/pulse.cio.gov/live/scan/inspect.csv">CSV</a>
+				Domain Inspect Results - <a href="https://s3-us-gov-west-1.amazonaws.com/cg-4adefb86-dadb-4ecf-be3e-f1c7b4f6d084/live/scan/inspect.csv">CSV</a>
```

Comment:
It would be better to use `meta.yml`, which should be possible in the template -- I just didn't bother doing the refactor here yet.